examples/lib/stories/components/components.dart

```diff
@@ -28,5 +24,11 @@ void addComponentsStories(Dashbook dashbook) {
       'Debug',
       (_) => GameWidget(game: DebugGame()),
       codeLink: baseLink('components/debug.dart'),
+    )
+    ..add(
+      'Game-in-game',
```

Comment:
Oh, you should record this example running and showcase it on the PR description.

Reply:
Done!